var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -188,35 +205,32 @@ def check_install(self):
             # TODO: Automate these path and library settings
             cc('-c', "-I%s" % join_path(spec.prefix, "include"), "check.c")
             cc('-o', "check", "check.o",
-               "-L%s" % join_path(spec.prefix, "lib"),
-               "-L%s" % join_path(spec.prefix, "lib64"),
```

Comment:
@eklee15: it looks like when you rebase you just kept your version of everything.  Can you do the rebase again but merge with upstream changes instead of overwriting?